pkg/ccl/changefeedccl/cdcevent/event.go

```diff
@@ -0,0 +1,491 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+package cdcevent
```

Comment:
Give us a doc.go and a comment explaining how this package fits into the puzzle

Reply:
Done.